graylog2-web-interface/src/components/events/events/BulkActions.test.tsx

```diff
@@ -27,7 +27,7 @@ import type { Event } from 'components/events/events/types';
 
 jest.mock('hooks/usePluginEntities');
 jest.mock('components/common/EntityDataTable/hooks/useSelectedEntities');
-
+jest.mock('components/events/events/hooks/useSendEventActionTelemetry', () => () => jest.fn());
```

Comment:
We can just shorten it like this:
```suggestion
jest.mock('components/events/events/hooks/useSendEventActionTelemetry');
```